### DISCUSSION TITLE
Files wrongly reported as untracked?

### DISCUSSION THREAD

**Discussion by stumpylog** [immich-app/immich#7435] [Q&A] [answered]
After updating to 1.95.1, approximately half my images are reported as being "untracked" on the repair page.  However, I can locate the images in question from the main timeline and they are viewable and downloadable.

Taking a single image as an example:

The repair page lists `/usr/src/app/upload/library/me/2023/01-January/2023-01-14/PXL_20230114_222814182.jpg` as untracked.
The image page lists the file as `upload/library/me/2023/01-January/2023-01-14/PXL_20230114_222814182.jpg`
The image downloads as PXL_20230114_222814182.jpg and the SHA matches the stored image on the server.

So, this appears to be an issue, right?

**Comment by Allram** [comment]
I have the same problem. Approx 16k of 84k images are reported as untracked, and same as you, they are reachable in the UI and in albums.

**Comment by bo0tzz** [comment, marked as answer]
Converted to an issue -> #7436
